src/Compilers/VisualBasic/Portable/Symbols/Retargeting/RetargetingFieldSymbol.vb

```diff
@@ -206,6 +206,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Retargeting
             Return _underlyingField.GetConstantValue(inProgress)
         End Function
 
+        Public Overrides ReadOnly Property IsRequired As Boolean
+            Get
+                Return _underlyingField.IsRequired
```

Comment:
Would Aleksey's comment in `RetargetingNamedTypeSymbol` apply here as well? If this is always false, consider asserting so. Also applies to other retargeting symbols.

Reply:
For clarification, the comment I was referring to was:
> I think all retargeting symbols are wrappers around symbols from VB module. So, they cannot have any required member "things". Instead of having "dead" code, let's assert that underlying symbols return false and empty sets form the new APIs and explicitly return false and empty sets.   

The question here is, can we add an assertion that `_underlyingField.IsRequired` is False?

Reply:
Tightened the asserts and added a test that adds a required member to a base type that requires retargeting a VB symbol to include required members.